### DISCUSSION TITLE
Save window configuration

### DISCUSSION THREAD

**Discussion by XInTheDark** [rxhanson/Rectangle#1012] [Ideas]
Please make it possible to bind the configuration of a certain window to a certain keyboard shortcut.

**Comment by Aaron-Rumpler** [comment]
[Rectangle Pro](https://rectangleapp.com/pro) supports this.

**Comment by rxhanson** [comment]
Indeed, Rectangle Pro does support saving and restoring window layouts (thanks @Aaron-Rumpler). I'm a little unclear what @XInTheDark is asking for here. 

> bind the configuration of a certain window to a certain keyboard shortcut

Window identification does not persist when it is closed and there are limitations to distinguishing windows. If you're looking to always move, say, a Chrome window with a specific set of tabs, then there's not really a way for Rectangle/Pro to distinguish that window from another Chrome window with another set of tabs. 

If you're just looking to save off window layouts and apply them, then Rectangle Pro will do this, and if you have multiple windows of the same app then the app will simply apply each window size/position in the order that they are stacked (z-index).

This type of behavior is outside of the scope of what I would consider adding into Rectangle, which is intentionally slim.
